amethyst_ui/src/event.rs

```diff
@@ -48,6 +48,10 @@ pub enum UiEventType {
         /// The entity on which the dragged object was dropped.
         dropped_on: Entity,
     },
+    /// When the value of an `Editable` Ui element has changed.
+    Change,
+    /// When an `Editable` Ui element has been committed.
+    Commit,
```

Comment:
By "commit" you mean something like "confirm" or "submit"? I think this is not very obvious from the name, maybe submit confirm would be better since that's what's generally used in "form speak" (unless there's some other convention here that I'm not aware of).

Reply:
Precisely that, yeah. I avoided "submit" simply because it is generally associated with the larger concept of "forms", which were out of scope for this. I figured a submit generally implies multiple fields, but I'd be happy to change it if the consensus is that "submit" is more clear.

Reply:
I like it a lot better now, thanks!